src/VoltageCalibration.cc

```diff
@@ -1058,110 +1053,47 @@ double * mattak::applyVoltageCalibration (int N, const int16_t * in, double * ou
     return 0;
   }
 
-  bool is2ndHalfWindows;
-  int nwindows = N / mattak::k::radiant_window_size;
-  int isamp;
-  int i = 0;
-  int j;
+  int nSamples_wf = N;
```

Comment:
why would you do this? you can rename the variable without breaking user code, right?

Reply:
I know I could've just renamed the "N" to a new name, but I just thought that the "N" matches the other two arguments the **in** and the **out** better (also there are N's in the python binding function in the bottom of the script, we could change all of them though). Originally in the function `double * mattak::applyVoltageCalibration()` it was `int nwindows = N / mattak::k::radiant_window_size;` so I just kept the format as the original, but again we could change them.

Reply:
I would change them